group.py

```diff
@@ -430,8 +440,18 @@ def api_group_update(ctx, group_name, property_name, property_value):
     :param group_name:     Name of the group to update property of
     :param property_name:  Name of the property to update
     :param property_value: Value of the property to update
+
+    :returns: Dict holding process status and process status info
     """
-    ctx.uuGroupModify(group_name, property_name, property_value, '', '')
+    try:
+        response = ctx.uuGroupModify(group_name, property_name, property_value, '', '')['arguments']
+        status = response[3]
+        message = response[4]
+        return {'proc_status': status,
```

Comment:
We should use the API responses instead of custom dicts. Inside the `try` we can probably return 'api.Result.ok()' is status is 1. If the status is 0 or not 1 we can return the correct error message with api.Error. The exception returns all further unhandled errors.